Tests/Contour.Common.Tests/PatternsSpecs.cs

```diff
@@ -287,7 +287,7 @@ public void should_accept_message()
                     "Body");
 
                 Mock<IDelivery> deliveryMock = new Mock<IDelivery>();
-                BusProcessingContext.Current = new BusProcessingContext(deliveryMock.Object);
+                BusProcessingContext.Current = new BusProcessingContext(deliveryMock.Object, null);
```

Comment:
Looks like we should provide a mock here and enforce bus parameter validation inside Bus Processing Context as null does not make any sense for the client code. In other words how was this delivery delivered? The answer imho should be  "by bus" :)